api/admin.py

```diff
@@ -3,7 +3,7 @@
 
 from django.contrib import admin
 
-from models import CSTeam, CSUser, HotoheteSettings
+from api.models import CSTeam, CSUser, HotoheteSettings
```

Comment:
models works, but api.models is the correct path, right?

Reply:
models is a relative path, it is correct, but it can be conflicts between local modules and "root" modules. So, it is recommended to use always references from root to avoid it.

Reply:
Absolute path is the path :D